app/views/shared/_approval_status.html.erb

```diff
@@ -0,0 +1,9 @@
+<%- if @cart.flow == 'linear' %>
+  <%= render partial: "shared/linear_approval_status" %>
+<%- else %>
+  <div class="col-md-4  col-xs-12 communicart_description">
+    <div class="purchase-status">
+      <h5 class="<%= @cart.status %>"><%= @cart.display_status %></h5>
+    </div>
+  </div>
+<%- end %>
```

Comment:
Created a story with an idea to avoid this split: https://www.pivotaltracker.com/story/show/85405198